data-prepper-core/src/main/java/com/amazon/dataprepper/pipeline/Pipeline.java

```diff
@@ -76,13 +76,15 @@ public Pipeline(
         this.sinks = sinks;
         this.prepperThreads = prepperThreads;
         this.readBatchTimeoutInMillis = readBatchTimeoutInMillis;
-        int coreThreads = sinks.size() + prepperThreads; //TODO We may have to update this after benchmark tests
-        this.prepperSinkExecutorService = PipelineThreadPoolExecutor.newFixedThreadPool(coreThreads,
+        this.prepperExecutorService = PipelineThreadPoolExecutor.newFixedThreadPool(prepperThreads,
                 new PipelineThreadFactory(format("%s-process-worker", name)), this);
+
+        this.sinkExecutorService = PipelineThreadPoolExecutor.newFixedThreadPool(sinks.size(),
+                new PipelineThreadFactory(format("%s-process-worker", name)), this);
```

Comment:
assuming it is intentional to name sink threads also as `...process-worker...`

Reply:
Was not intentional, good point. Will changes names to be more indicative of preppers/sinks.